main.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/hashicorp/terraform-ls/commands"
 )
 
+//go:generate go run schemas/gen/gen.go
```

Comment:
Any reason? I can move it, but I'll be making a near dummy file, and it may mess with the relative paths in gen.go

Reply:
AFAIK it's a good practice (see e.g. Terraform core) to keep generate comments scoped to the relevant package. Technically we could also put it into `internal/terraform/rootmodule` as that's where we use it, but `schemas` is just easier to avoid all the directory traversal.

The main point is that once you have more than 1 generate annotation, you can run each individually and still retain the ability to run all via `go generate ./...`.

Reply:
Less related to the generate comment - I think the `schemas` package should be somewhere under `internal` as we don't want other people to be importing it.